mods/d2k/maps/shellmap/shellmap.lua

```diff
@@ -16,4 +16,5 @@ WorldLoaded = function()
 	atreides = Player.GetPlayer("Atreides")
 
 	InsertHarvester()
+	Media.PlayMusic("score")
```

Comment:
I intentionally added those as different test cases on how to use the function. The problem with looping songs is also that they can become annoying. Only the Tiberian Dawn one was subtle enough for my taste, but I don't have a strong preference here. I just want this finally merged.